src/huggingface_hub/file_download.py

```diff
@@ -753,6 +886,10 @@ def hf_hub_download(
         local_files_only (`bool`, *optional*, defaults to `False`):
             If `True`, avoid downloading the file and return the path to the
             local cached file if it exists.
+        legacy_cache_layout (`bool`, *optional*, defaults to `False`):
+            If `True`, uses the legacy file cache layout i.e. just call `hf_hub_url`
```

Comment:
```suggestion
            If `True`, uses the legacy file cache layout i.e. just call [`hf_hub_url`]
```
Link to `hf_hub_url`?